crates/astria-sequencer/src/sequencer.rs

```diff
@@ -28,9 +30,10 @@ impl Sequencer {
     pub async fn run_until_stopped(config: Config) -> Result<()> {
         let genesis_state =
             GenesisState::from_path(config.genesis_file).context("failed reading genesis state")?;
-        let storage = penumbra_storage::TempStorage::new()
+        let db_path = PathBuf::from(config.db_datadir);
+        let storage = penumbra_storage::Storage::load(db_path.clone())
```

Comment:
Can you check for the existence of a file and report whether a db will be created or opened? This isn't ideal because it should happen inside the `Storage::load` function, but it is what it is.